src/components/MenuBar.vue

```diff
@@ -41,17 +41,20 @@ function logout() {
           contain
         ></v-img>
       </router-link>
+
       <v-toolbar-title class="title">
         {{ title }}
       </v-toolbar-title>
       <v-spacer></v-spacer>
-      <v-btn class="mx-2" :to="{ name: 'recipes' }"> Recipes </v-btn>
+
+      <v-btn v-if="user == null" class="mx-2" :to="{ name: 'stories' }">Stories</v-btn>
```

Comment:
should this check for `user == null` be here?  Should we show stories to a user not logged in?